### PR TITLE
Initial support for testing with a live-iso

### PR DESCRIPTION
**Pull request by hardys**
Add initial support for downloading an FCOS iso, related to
development of https://github.com/metal3-io/metal3-docs/pull/150 and testing of https://github.com/metal3-io/baremetal-operator/pull/754

### CODE REVIEW

**Comment by fmuyassarov** [comment]
Ubuntu integration job is failing when running `kubeadm init`. It somehow could be related to the latest Docker version that was released 2 days ago. So, if the job fails on this PR, it might be due the common issue. Will update here once we have fixed it.

**Comment by fmuyassarov** [comment]
CI should be back to normal now. Triggering integration tests.
/test-integration
/test-centos-integration

File: lib/images.sh [hunk 1]
```diff
@@ -6,8 +6,11 @@ if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
   export IMAGE_NAME=${IMAGE_NAME:-UBUNTU_20.04_NODE_IMAGE_K8S_v1.19.3.qcow2}
   export IMAGE_LOCATION=${IMAGE_LOCATION:-https://artifactory.nordix.org/artifactory/airship/images/k8s_v1.19.3/}
 elif [[ "${IMAGE_OS}" == "FCOS" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-32.20200923.2.0-openstack.x86_64.qcow2.xz}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20200923.2.0/x86_64/}
+  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-32.20201104.3.0-openstack.x86_64.qcow2.xz}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20201104.3.0/x86_64/}
+elif [[ "${IMAGE_OS}" == "FCOS-ISO" ]]; then
+  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-33.20201201.2.1-live.x86_64.iso}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/33.20201201.2.1/x86_64}
```
Comment (fmuyassarov) [inline lib/images.sh:13]:
```suggestion
  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/33.20201201.2.1/x86_64/}
```

File: lib/images.sh [hunk 2]
```diff
@@ -6,8 +6,11 @@ if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
   export IMAGE_NAME=${IMAGE_NAME:-UBUNTU_20.04_NODE_IMAGE_K8S_v1.19.3.qcow2}
   export IMAGE_LOCATION=${IMAGE_LOCATION:-https://artifactory.nordix.org/artifactory/airship/images/k8s_v1.19.3/}
 elif [[ "${IMAGE_OS}" == "FCOS" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-32.20200923.2.0-openstack.x86_64.qcow2.xz}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20200923.2.0/x86_64/}
+  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-32.20201104.3.0-openstack.x86_64.qcow2.xz}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20201104.3.0/x86_64/}
```
Comment (fmuyassarov) [inline lib/images.sh:10]: Are we missing something in the image path ?

```
$ wget https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20201104.3.0/x86_64/fedora-coreos-32.20201104.3.0-openstack.x86_64.qcow2.xz

--2020-12-20 13:14:27--  https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20201104.3.0/x86_64/fedora-coreos-32.20201104.3.0-openstack.x86_64.qcow2.xz
Resolving builds.coreos.fedoraproject.org (builds.coreos.fedoraproject.org)... 205.251.219.79, 205.251.219.37, 205.251.219.127, ...
Connecting to builds.coreos.fedoraproject.org (builds.coreos.fedoraproject.org)|205.251.219.79|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2020-12-20 13:14:28 ERROR 403: Forbidden.

```

File: lib/images.sh (hunk 1 above)
Comment (hardys) [reply lib/images.sh:13]: @fmuyassarov thanks for the suggestion, but I don't think the trailing slash is needed:

```
./01_prepare_host.sh:    curl --insecure --compressed -O -L "${IMAGE_LOCATION}/${IMAGE_NAME}"
```

It's also used inconsistently, so we should perhaps consider removing it for existing images in a separate PR?

File: lib/images.sh (hunk 2 above)
Comment (hardys) [reply lib/images.sh]: Ah thanks, I think the update to the openstack image was a mistake, now reverted to the previous version and we can perhaps update it in a future commit.

File: lib/images.sh (hunk 1 above)
Comment (fmuyassarov) [reply lib/images.sh:13]: You are right.  
+1 for removing extra slash in another PR.

**Pull request merged**